docs/pages/includes/install-linux.mdx

```diff
@@ -1,112 +1,116 @@
-        $ cd teleport
-        $ sudo ./install
-        ```
   </TabItem>
+  <TabItem label="Tarball">
+
+    ```code
+    $ curl https://get.gravitational.com/teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz.sha256
+    # <checksum> <filename>
+    $ curl -O https://get.gravitational.com/teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz
+    $ shasum -a 256 teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz
+    # Verify that the checksums match
+    $ tar -xzf teleport-v(=teleport.version=)-linux-amd64-bin.tar.gz
+    $ cd teleport
+    $ sudo ./install
+    ```
 
-  <TabItem label="Debian/Ubuntu Legacy (DEB)">
-        ```code
-        # Using this APT repo may result in breaking upgrades upon "apt upgrade" as all major versions will be
-        # published under the same component. We recommend following the instructions in the
-        # "Debian/Ubuntu (DEB)" tab instead.
-        # Download Teleport's PGP public key
-        $ sudo curl https://deb.releases.teleport.dev/teleport-pubkey.asc \
-          -o /usr/share/keyrings/teleport-archive-keyring.asc
-        # Add the Teleport APT repository
-        $ echo "deb [signed-by=/usr/share/keyrings/teleport-archive-keyring.asc] https://deb.releases.teleport.dev/ stable main" \
-        | sudo tee /etc/apt/sources.list.d/teleport.list > /dev/null
-
-        $ sudo apt-get update
-        $ sudo apt-get install teleport
-        ```
   </TabItem>
 
-  <TabItem label="Amazon Linux 2/RHEL Legacy (RPM)">
-        ```code
-        $ sudo yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
-        $ sudo yum install teleport
+  <TabItem label="ARMv7 (32-bit)">
+
+      ```code
+      $ curl https://get.gravitational.com/teleport-v(=teleport.version=)-linux-arm-bin.tar.gz.sha256
+      # <checksum> <filename>
+      $ curl -O https://get.gravitational.com/teleport-v(=teleport.version=)-linux-arm-bin.tar.gz
+      $ shasum -a 256 teleport-v(=teleport.version=)-linux-arm-bin.tar.gz
+      # Verify that the checksums match
+      $ tar -xzf teleport-v(=teleport.version=)-linux-arm-bin.tar.gz
+      $ cd teleport
+      $ sudo ./install
+      ```
 
-        # Optional:  Using DNF on newer distributions
-        # $ sudo dnf config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
-        # $ sudo dnf install teleport
+  </TabItem>
+  <TabItem label="ARM64/ARMv8 (64-bit)">
+    ```code
+    $ curl https://get.gravitational.com/teleport-v(=teleport.version=)-linux-arm64-bin.tar.gz.sha256
+    # <checksum> <filename>
+    $ curl -O https://get.gravitational.com/teleport-v(=teleport.version=)-linux-arm64-bin.tar.gz
+    $ shasum -a 256 teleport-v(=teleport.version=)-linux-arm64-bin.tar.gz
+    # Verify that the checksums match
+    $ tar -xzf teleport-v(=teleport.version=)-linux-arm64-bin.tar.gz
+    $ cd teleport
+    $ sudo ./install
         ```
   </TabItem>
+  <TabItem label="Debian/Ubuntu Legacy (DEB)">
```

Comment:
Do we even need to include these legacy repos at all?

Reply:
I have no idea. These were already here, I just adjusted the indentation on these blocks.